ui/src/app/pages/signed-in/nav-bar.tsx

```diff
@@ -151,12 +150,7 @@ export const NavBar = () => {
         )}
       </div>
       <Breadcrumb />
-      {window.location.pathname !== '/access-renewal' && (
-        <AccessRenewalNotificationMaybe />
-      )}
-      {window.location.pathname !== '/data-access-requirements' && (
-        <LoginGovIAL2NotificationMaybe />
-      )}
+      <AccessRenewalNotificationMaybe />
```

Comment:
It's still "maybe" because it only renders when the user needs to do annual renewal.